### ISSUE TITLE
🐞 [BUG]: Translations between nuxt and data-experience module are not well merged

### ISSUE THREAD

**Issue by fquellec** [hestiaAI/hestialabs-experiences#1175]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Current Behavior

if we add a locale key to data-experience/locales, it is correctly used when running the module with `npm run dev`.
But once the data-experience is run under the nuxt app, the locales' keys are not found anymore. 

Example with Tiktok experience, Login History Tab: 
 - data-experience - npm run dev 
   <img width="417" alt="image" src="https://user-images.githubusercontent.com/17878016/202417437-9f098f76-53c7-43e7-af45-a887d6a17058.png">
 -  nuxt-app - npm run dev 
   <img width="381" alt="image" src="https://user-images.githubusercontent.com/17878016/202418051-92284ae6-58ee-4014-b9a7-25d5bbc4608d.png">

### Expected Behavior

if the key is not found in the nuxt locales we should use the data-experience locales

### Steps to Reproduce

1. npm run dev on experiences
2. Go go TikTok experience
3. Login history tab
4. look at the bottom text of the last graph

### Web Sites

digipower.academy

### Browser

Chromium

**Comment by valentinoli** [comment]
yes, this is really some unfinished business related to modularization

We should start with filtering both data-experience/locales and experiences/locales 

and then figure out how to include data-experience/locales in the Nuxt app
